libs/sdk-ui-charts/src/charts/headline/internal/utils/calculateCustomHeight.ts

```diff
@@ -34,16 +34,16 @@ export function calculateHeadlineHeightFontSize(
     if (!secondaryItem) {
         height = clientHeight < HEIGHT.NORMAL ? clientHeight : HEIGHT.NORMAL;
         fontSize =
-            clientHeight === HEIGHT.SMALLEST
+            clientHeight <= HEIGHT.SMALLEST
```

Comment:
Please, rewrite this double ternary operator to a more readable way (for example if statement).